Makefile

```diff
@@ -159,30 +141,40 @@ inttestbin:
 debuginttest1: inttestbin
 	cd pkg/jx/cmd && dlv --listen=:2345 --headless=true --api-version=2 exec ../../../build/jx-inttest -- -test.run $(TEST)
 
-install: $(GO_DEPENDENCIES) version
+install: $(GO_DEPENDENCIES)
 	GOBIN=${GOPATH}/bin $(GO) install $(BUILDFLAGS) cmd/jx/jx.go
 
 fmt:
 	@FORMATTED=`$(GO) fmt ./...`
 	@([[ ! -z "$(FORMATTED)" ]] && printf "Fixed unformatted files:\n$(FORMATTED)") || true
 
-arm: version
+linux:
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux GOARCH=amd64 $(GO) build $(BUILDFLAGS) -o build/linux/jx cmd/jx/jx.go
+
+arm:
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux GOARCH=arm $(GO) build $(BUILDFLAGS) -o build/$(NAME)-arm cmd/jx/jx.go
 
-win: version
+win:
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=windows GOARCH=amd64 $(GO) build $(BUILDFLAGS) -o build/$(NAME).exe cmd/jx/jx.go
 
-win32: version
+win32:
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=windows GOARCH=386 $(GO) build $(BUILDFLAGS) -o build/$(NAME)-386.exe cmd/jx/jx.go
 
-darwin: version
+darwin:
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=darwin GOARCH=amd64 $(GO) build $(BUILDFLAGS) -o build/darwin/jx cmd/jx/jx.go
 
 # sleeps for about 30 mins
 sleep:
 	sleep 2000
 
-release: check
+set-release-version:
+    # ensure we have the latest tag
+	@git fetch --all -q 2>/dev/null
+	$(eval VERSION = $(shell git describe --abbrev=0 --tags 2>/dev/null | sed 's/^v//' ))
+	@echo Release version: $(VERSION)
+
+.PHONY: release
+release: set-release-version check
```

Comment:
@jstrachan - for the release we depend on a new `set-release-version` which calls `git fetch` and dynamically updates VERSION using eval.